<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportLasWizardPage</class>
 <widget class="QWizardPage" name="ImportLasWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import .las file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="upperGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="selectalldefinedRadioButton">
        <property name="text">
         <string>Select all defined</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectallRadioButton">
        <property name="text">
         <string>Select all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectnoneRadioButton">
        <property name="text">
         <string>Select none</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="upperHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="logsTableWidget"/>
   </item>
   <item>
    <widget class="QGroupBox" name="lowerGroupbox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="viewfilePushButton">
        <property name="text">
         <string>View file </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="lowerHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
